README.md

```diff
@@ -1,111 +1,50 @@
+[![Hackage][hk-img-core]][hk-core]
+[![Stackage LTS][lts-img-core]][lts-core]
 [![MPL-2.0 license](https://img.shields.io/badge/license-MPL--2.0-blue.svg)](https://github.com/kowainik/co-log/blob/main/LICENSE)
 
+`co-log-core` is a lightweight package that provides core types and functions to 
+work with the @LogAction@ data type which is both simple and powerful.
 
-|                   |                                    |                                           |                                                       |
-| :------------     | :--------------------------------- | :---------------------------------------- | :---------------------------------------------------- |
-| `co-log-core`     | [![Hackage][hk-img-core]][hk-core] | [![Stackage LTS][lts-img-core]][lts-core] | [![Stackage Nightly][nightly-img-core]][nightly-core] |
-| `co-log`          | [![Hackage][hk-img]][hk]           | [![Stackage LTS][lts-img]][lts]           | [![Stackage Nightly][nightly-img]][nightly]           |
-| `co-log-polysemy` | [![Hackage][hk-img-ps]][hk-ps]     | [![Stackage LTS][lts-img-ps]][lts-ps]     | [![Stackage Nightly][nightly-img-ps]][nightly-ps]     |
+## How to use
 
-`co-log` is a composable and configurable logging framework. It
-combines all the benefits of Haskell idioms to provide a reasonable
-and convenient interface. Though it uses some advanced concepts in its
-core, we are striving to provide beginner-friendly API. The library
-also contains complete documentation with a lot of beginner-friendly
-examples, explanations and tutorials to guide users. The combination
-of a pragmatic approach to logging and fundamental Haskell abstractions
-allows us to create a highly composable and configurable logging
-framework.
+`co-log-core` is compatible with the following GHC
+versions - [supported versions](https://matrix.hackage.haskell.org/#/package/co-log-core)
 
-If you're interested in how different Haskell typeclasses are used to
-implement core functions of `co-log`, you can read the following blog
-post which goes into detail about internal implementation specifics:
+In order to start using `co-log-core` in your project, you
+will need to set it up with these steps:
 
-* [co-log: Composable Contravariant Combinatorial Comonadic Configurable Convenient Logging](https://kowainik.github.io/posts/2018-09-25-co-log)
+1. Add the dependency on `co-log-core` in your project's
+   `.cabal` file. For this, you should modify the `build-depends`
+   section by adding the name of this library. After the adjustment,
+   this section could look like this:
 
-`co-log` is also modular on the level of packages. We care a lot about a
-low dependency footprint so you can build your logging only on top of
-the minimal required interface for your use-case. This repository contains
-the following packages:
+   ```haskell
+   build-depends: base ^>= LATEST_SUPPORTED_BASE
```

Comment:
It looks like this section misses the dependencies on `co-log-core` 🤔

Reply:
According to cabal, this package seems not having any more dependencies to be mentioned in documentation. If I have mistaken then please correct me.

Reply:
Well, `co-log-core` is not a part of `base`, it's a separate package. And if you want to use it in your own package, you need to add the library to your dependencies in the `.cabal` file.

Reply:
Shall i modify build-depends section like this?
```
 build-depends:       base ^>= LATEST_SUPPORTED_BASE
                          ,       co-log-core ^>= LATEST_VERSION
```

Reply:
Yeah, something like this should be fine 👍🏻